internal/backend/crypto/gpg/gpgconf/version_test.go

```diff
@@ -44,7 +44,7 @@ func TestSort(t *testing.T) {
 
 			sort.Sort(byVersion(tc.in))
 
-			require.Equal(t, len(tc.in), len(tc.out))
+			require.Len(t, len(tc.in), len(tc.out))
```

Comment:
This looks suspicious, maybe you meant 
```suggestion
			require.Len(t, tc.out, len(tc.in))
```

See https://pkg.go.dev/github.com/stretchr/testify/require#Len